tests/units/physics/test_kinetic.py

```diff
@@ -13,3 +14,18 @@ def test_kinetic_energy_shape():
     kinetic_energies = kinetic_energy_fn(None, x)
 
     assert kinetic_energies.shape == (x.shape[0],)
+
+
+def test_hubbard_kinetic_energy_shape():
+    """Check that the vmapped shape is correct."""
+
+    f,_ = make_dummy_log_f() #not in log domain
+    x = jnp.array([[0,1,3]])
+    side_length=4
+
+    kinetic_energy_fn = physics.kinetic.create_hubbard_kinetic_energy(f,side_length)
+    kinetic_energies = kinetic_energy_fn(None, x)
+
+    assert kinetic_energies.shape == (x.shape[0],)
+
+
```

Comment:
I think in this file, it could make sense to test the value of the local kinetic energy, since for just three particles the exact formula is quite simple.